searchfingerprint

```diff
@@ -40,7 +40,7 @@ while [ "${*}" != "" ] ; do
     shift
 
     #Confirm input is a video file
-    if [[ -z $(file -Ib "${INPUT}" | grep video) ]] ; then
+    if [[ -z $(file -i "${INPUT}" | grep video) ]] ; then
```

Comment:
This breaks for me. Maybe using ffmpeg is safer.
```
file -i TEST.mov 
/Users/daverice/Desktop/TEST.mov: regular file
```
vs
```
file -Ib TEST.mov 
video/quicktime; charset=binary
```

Reply:
>Maybe using ffmpeg is safer.

+1